tools/flaky-test-reporter/main.go

```diff
@@ -48,19 +49,22 @@ func main() {
 		log.Fatalf("config cannot be created: '%v'", err)
 	}
 
-	if nil != dryrun && true == *dryrun {
+	if true == *dryrun {
 		log.Printf("running in [dry run mode]")
 	}
+	if true == *skipReport {
+		log.Printf("--skip-report provided, skipping Github and Slack report")
+	}
 
 	if err := prow.Initialize(*serviceAccount); nil != err { // Explicit authenticate with gcs Client
 		log.Fatalf("Failed authenticating GCS: '%v'", err)
 	}
 	gih, err := Setup(*githubAccount)
```

Comment:
You could skip setting up GitHub and Slack clients if `!*skipReport`. Up to you.

Reply:
done